src/main/java/AppointmentPage.java

```diff
@@ -116,14 +116,13 @@ public AppointmentPage prevMount(){
         return this;
     }
 
-    public AppointmentPage clickCommentTF(String comment){
-        //clickCommentTextField.click();
-        clickCommentTextField.sendKeys(comment);
-       return this;
+    public AppointmentPage commentTextField(String comment){
+        commentTextField.sendKeys(comment);
+        return this;
     }
```

Comment:
Incorrect name of the method: "clickCommentTF"
It should be putCommentTF, or enterCommentTF

Reply:
fixed